internal/controllers/s3userclaim/handler.go

```diff
@@ -129,3 +129,21 @@ func (r *Reconciler) initVars(req ctrl.Request) {
 
 	r.s3UserName = fmt.Sprintf("%s.%s", req.Namespace, req.Name)
 }
+
+func cephSubUserFullIdMaker(cephUserFullId string, subUser string) string {
+	return fmt.Sprintf("%s:%s", cephUserFullId, subUser)
+}
+
+func subUserSecretNameMaker(s3UserClaimName string, subUser string) string {
+	return fmt.Sprintf("%s-%s", s3UserClaimName, subUser)
+}
+
+func subUserNameExtractor(cephSubUserFullId string) (string, error) {
+	parts := strings.Split(cephSubUserFullId, ":")
+	if len(parts) == 2 {
+		// return the last part since it's the subUser name
```

Comment:
Unnecessary. Can be deduced from the function name.

Reply:
Done